### PR TITLE
fix: stripe selected currency

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

- Change behaviour in event type app card, so initially we will have default values on the form context.
- Undo required changed in test that required to always select a currency because it didn't had any defaul value initially.

Fixes # (issue)
Follow up for alby and payment apps

- [X] Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

File: apps/web/playwright/fixtures/users.ts [hunk 1]
```diff
@@ -556,8 +556,6 @@ export async function apiLogin(
 export async function setupEventWithPrice(eventType: Pick<Prisma.EventType, "id">, page: Page) {
   await page.goto(`/event-types/${eventType?.id}?tabName=apps`);
   await page.locator("div > .ml-auto").first().click();
-  await page.locator(".text-black > .bg-default > div > div:nth-child(2)").first().click();
-  await page.getByTestId("select-option-usd").click();
```
Comment (alannnc) [inline apps/web/playwright/fixtures/users.ts:560]: This was selecting currency first select option

File: packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx [hunk 2]
```diff
@@ -25,8 +25,8 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
   const currency = getAppData("currency");
   const [selectedCurrency, setSelectedCurrency] = useState(
     currencyOptions.find((c) => c.value === currency) || {
-      label: "",
-      value: "",
+      label: currencyOptions[0].label,
+      value: currencyOptions[0].value,
```
Comment (alannnc) [inline packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx:29]: Set first currencyOption as the initial default value (USD)

File: packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx [hunk 3]
```diff
@@ -46,6 +46,16 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
...
       .trim();
+
+  useEffect(() => {
+    if (!getAppData("currency")) {
+      setAppData("currency", currencyOptions[0].value);
+    }
+    if (!getAppData("paymentOption")) {
+      setAppData("paymentOption", paymentOptions[0].value);
+    }
+  }, []);
```
Comment (alannnc) [inline packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx:57]: When mounting event type app card, we set initial values that will only be saved after submit.

**Review by zomars — APPROVED** [review]
Nice work @alannnc 🙏🏽

**Pull request merged**